packages/client/src/createTRPCClientProxy.ts

```diff
@@ -91,24 +97,41 @@ const clientCallTypeMap: Record<
   subscribe: 'subscription',
 };
 
+export type CreateTRPCProxyClient<TRouter extends AnyRouter> =
+  inferRouterProxyClient<TRouter> & {
+    /**
+     * the TRPC runtime
+     */
+    runtime: TRPCClientRuntime;
+  };
+
```

Comment:
I think the `{ runtime: TRPCClientRuntime }` part should come before `inferRouterProxyClient` so that if users have procedures called `runtime`, the type will still show correctly and match runtime behavior.